keymanapp/keyman
# chore(android,linux,windows): Check in crowdin strings for Bura-Pabir

This checks in the crowdin strings for Bura-Pabir (language code bwr-NG) though some of the platforms automatically use bwr-NT.

TODO: reun iOS project through XCode

## User Testing
For all the platforms, load the PR build, set the UI to "Bura-Pabir", and verify the UI is in Bura-Pabir

* **TEST_ANDROID** - Tests Bura-Pabir UI for Android
1. Load the PR build in an Android 5.0 emulator (SDK 21)
2. In the Keyman app, go to Settings --> Display Language --> Bura-Pabir
3. Verify the UI is in Bura-Pabir

* **TEST_IOS** - Tests Bura-Pabir UI for iOS [Skip since not available]
1. Start the app
2. Change the UI to Bura-Pabir locale
3. Verify the app refreshes with Bura-Pabir strings

* **TEST_WINDOWS** - Tests Bura-Pabir UI for Windows

* **TEST_LINUX** - Tests Bura-Pabir UI for Linux
1.  start Keyman configuration with: `LANGUAGE=bwr_NT km-config`
